config/.codespell-whitelist

```diff
@@ -1,16 +1 @@
-uint
-ith
-nd
-mitre
-readded
-crate
-developper
-ist
-iam
-espace
-acn
-ende
-sting
-complies
-shiping
-furter
+MRnew
```

Comment:
I don't know why you wold be overriding this file

Reply:
This is related to the EIP draft and I thought to have spotted some malicious entries there.